index.html

```diff
@@ -0,0 +1,292 @@
+          </p>
+          <ul class="tags">
+            <li>css</li>
+            <li>html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="button-works">See Project</button>
+        </div>
+      </div>
+
+      <div class="works-list">
+        <div class="right-block">
+          <h4>Multi-Post Stories</h4>
+          <p class="work-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="tags">
+            <li>css</li>
+            <li>html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="button-works">See Project</button>
+        </div>
+        <div class="featured-img">
+          <img src="ImgPlaceholder.png" alt="Image placeholder" />
+        </div>
+      </div>
+
+      <div class="works-list">
+        <div class="featured-img">
+          <img src="ImgPlaceholder.png" alt="Image placeholder" />
+        </div>
+        <div class="right-block">
+          <h4>Multi-Post Stories</h4>
+          <p class="work-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="tags">
+            <li>css</li>
+            <li>html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="button-works">See Project</button>
+        </div>
+      </div>
+
+      <div class="works-list">
+        <div class="right-block">
+          <h4>Multi-Post Stories</h4>
+          <p class="work-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="tags">
+            <li>css</li>
+            <li>html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="button-works">See Project</button>
+        </div>
+        <div class="featured-img">
+          <img src="ImgPlaceholder.png" alt="Image placeholder" />
+        </div>
+      </div>
+    </section>
+    <!-- contact-us -->
+    <section id="contact-us">
+      <div class="contact-headline">
+        <h5>Let’s chat!</h5>
+        <p class="contact-text">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
```

Comment:
- [ ] Your let's chat heading should be on the middle of the page but it is on the left of it kindly make changes and make less spacing to it.
- [ ] I would suggest you to add same font-family and same color for you `p` tag.
### your design
![hisss](https://user-images.githubusercontent.com/81727988/171836023-80e4cf79-aca3-4267-8031-cf696c390a67.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/171836109-88fb7491-ae3b-4f67-ac72-4e531f387573.png)